sphinx/source/docs/user_guide/data.rst

```diff
@@ -44,9 +44,9 @@ highlighted in a second plot (:ref:`userguide_data_linked_selection`).
 At the most basic level, a |ColumnDataSource| is simply a mapping between column
 names and lists of data. The |ColumnDataSource| takes a ``data`` parameter which is a dict,
 with string column names as keys and lists (or arrays) of data values as values. If one positional
-argument is passed in to the |ColumnDataSource| initializer, it will be taken as ``data``. Once the
+argument is passed into the |ColumnDataSource| initializer, it will be taken as ``data``. Once the
```

Comment:
I actually think just "to" would work best

Reply:
Absolutely, I'll change that with my next commit!